contributions/api/utils/jsonutils.py

```diff
@@ -20,6 +20,24 @@ def remove_objectid_from_dataset(dataset):
 
     return dataset
 
+def convert_obejctid_from_dataset_json(dataset):
+    if "_id" in dataset:
+        if "$oid" in dataset["_id"]:
```

Comment:
@ywkim312, there is an error when trying to access `GET contributions/{{contributionId}}`

```
  File "rokwire-building-blocks-api/contributions/api/controllers/contributions.py", line 216, in get
    jsonutils.convert_obejctid_from_dataset_json(data_list[0])
  File "rokwire-building-blocks-api/contributions/api/utils/jsonutils.py", line 25, in convert_obejctid_from_dataset_json
    if "$oid" in dataset["_id"]:
TypeError: argument of type 'ObjectId' is not iterable
```
Please look into this when you get a chance. Thanks.

Reply:
Thanks for fixing the object ID error.

Now that it's possible to view `GET contributions/{{contributionId}}`, I see that `POST /contributions` is resulting in Talent and Capability ID not getting saved in the database. These are getting stored as null.

Reply:
Sorry, I forgot to test that endpoint after I updated the code. Fixed it, please try it again.

Reply:
So we want contribution back end make talent and capability id when POSTing. Okay, I changed that way.